core/model/src/test/java/org/eclipse/rdf4j/model/util/ModelsTest.java

```diff
@@ -445,4 +446,77 @@ public void testStripContextsSpecificContext() {
 		assertThat(graph2Stripped.size()).isEqualTo(model1.size());
 	}
 
+	@Test
+	public void testConvertReificationToRDFStar() {
+		Model reificationModel = RDFStarTestHelper.createRDFReificationModel();
+		Model referenceRDFStarModel = RDFStarTestHelper.createRDFStarModel();
+
+		Model rdfStarModel1 = Models.convertReificationToRDFStar(VF, reificationModel);
+		assertTrue("RDF reification conversion to RDF* with explicit VF, model-to-model",
+				Models.isomorphic(rdfStarModel1, referenceRDFStarModel));
+
+		Model rdfStarModel2 = Models.convertReificationToRDFStar(reificationModel);
+		assertTrue("RDF reification conversion to RDF* with implicit VF, model-to-model",
+				Models.isomorphic(rdfStarModel2, referenceRDFStarModel));
+
+		Model rdfStarModel3 = new TreeModel();
+		Models.convertReificationToRDFStar(VF, reificationModel, rdfStarModel3::add);
+		assertTrue("RDF reification conversion to RDF* with explicit VF, model-to-consumer",
+				Models.isomorphic(rdfStarModel3, referenceRDFStarModel));
+
+		Model rdfStarModel4 = new TreeModel();
+		Models.convertReificationToRDFStar(reificationModel, rdfStarModel4::add);
+		assertTrue("RDF reification conversion to RDF* with implicit VF, model-to-consumer",
+				Models.isomorphic(rdfStarModel4, referenceRDFStarModel));
```

Comment:
I believe you can use .equals on models and it uses isomorphic underneath.

Reply:
> I believe you can use .equals on models and it uses isomorphic underneath.

It does but I've just checked and we don't actually specify this in the API, as far as I can tell.